concourse/tasks/gcloud-get-secret.yaml

```diff
@@ -0,0 +1,30 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+params:
```

Comment:
you have 2 params sections in the task, and one has no value (I think you need "" for that?)

Reply:
In testing with fly it looks like no default value is an acceptable syntax.